src/app/pages/component-sidenav/_component-sidenav-theme.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@angular/material/theming';
+@import '../../../../node_modules/@angular/material/theming';
```

Comment:
You should be able to change all of these imports back; there was a short-lived bug in webpack's scss loader that made the `~` syntax not work that should be fixed with a fresh `npm install`